### PR TITLE
[MDS-5582] MS & Core - Allow more file types for report incident

### PR DESCRIPTION
**Pull request by isuru-aot**
## Objective 

[MDS-5582](https://bcmines.atlassian.net/browse/MDS-5582)

_Why are you making this change? Provide a short explanation and/or screenshots_

Screenshot 2023-12-21 145227
Screenshot 2023-12-21 145508

### CODE REVIEW

File: services/core-web/src/components/Forms/incidents/IncidentFormDocuments.tsx [hunk 1]
```diff
@@ -70,7 +70,7 @@ const IncidentFormDocuments: FC<IncidentFormDocumentsProps> = ({
                 id={INITIAL_INCIDENT_DOCUMENTS_FORM_FIELD}
                 name={INITIAL_INCIDENT_DOCUMENTS_FORM_FIELD}
                 labelIdle='<strong>Drag & Drop your files or <span class="filepond--label-action">Browse</span></strong><br>
-              <div>Accepted filetypes: .kmz, .doc, .docx, .xlsx, .pdf</div>'
+              <div>Accepted filetypes: .kmz, .doc, .docx, .xlsx, .pdf, .jpeg, .tiff, .hiec</div>'
```
Comment (taraepp) [inline services/core-web/src/components/Forms/incidents/IncidentFormDocuments.tsx]: Just noticed that .png isn't listed here! And should this include .msg or did that get taken out?

File: services/common/src/constants/fileTypes.js [hunk 2]
```diff
@@ -30,6 +30,11 @@ export const IMAGE = {
   ".jpg": "image/jpg",
   ".jpeg": "image/jpeg",
   ".png": "image/png",
+  ".jp2": "image/jp2",
+  ".tif": "image/tiff",
+  ".tiff": "image/tiff",
+  ".heic": "image/heic",
+  ".heif": "image/heif",
 };
```
Comment (matbusby-fw) [inline services/common/src/constants/fileTypes.js:39]: I think we'll want to create the message file type here as well to include in this update.  

Something like this:
```suggestion
export const MESSAGE = {
  ".msg": "application/vnd.ms-outlook",
};
```

File: services/core-web/src/components/Forms/incidents/IncidentFormDocuments.tsx (hunk 1 above)
Comment (isuru-aot) [reply services/core-web/src/components/Forms/incidents/IncidentFormDocuments.tsx]: .png and .msg was missed. Updated accordingly.

File: services/common/src/constants/fileTypes.js (hunk 2 above)
Comment (isuru-aot) [reply services/common/src/constants/fileTypes.js:39]: Thanks Done!

**Review by matbusby-fw — APPROVED** [review]
Nice!  🙌🏻

**Review by henryoforeh-dev — APPROVED** [review]
Nice!

**Pull request merged**